policy/modules/services/cron.te

```diff
@@ -499,6 +503,7 @@ corenet_udp_sendrecv_generic_if(system_cronjob_t)
 corenet_tcp_sendrecv_generic_node(system_cronjob_t)
 corenet_udp_sendrecv_generic_node(system_cronjob_t)
 corenet_udp_bind_generic_node(system_cronjob_t)
+corenet_tcp_connect_tor_port(system_cronjob_t)
```

Comment:
This seems to be out of place as it seems to be a use case specific to you, or at least, a very rare one.

Reply:
ok